docs/man/sat-jobstat.8.rst

```diff
@@ -0,0 +1,59 @@
+=============
+ SAT-JOBSTAT
+=============
+
+-------------------------------------------------------
+Jobstat - Check status of job(s) and its application(s)
+-------------------------------------------------------
+
+:Author: Hewlett Packard Enterprise Development LP.
+:Copyright: Copyright 2022 Hewlett Packard Enterprise Development LP.
+:Manual section: 8
+
+SYNOPSIS
+========
+
+**sat** [global-opts] **jobstat** [options]
+
+DESCRIPTION
+===========
+
+Jobstat is a command that allows a user or admin to access application and job
+data via the command line. The command provides a table showing the user the
+apid, jobid, user, state, and time-reported of all jobs on the system.
+
+OPTIONS
+=======
+
+These options must be specified after the subcommand.
+
+**-h, --help**
+        Print the help message for 'sat jobstat'.
+.. include:: _sat-format-opts.rst
```

Comment:
Please add an empty line between lines 31 and 32. As it currently exists, compiling the man pages gives a warning:
```
sat-jobstat.8.rst:32: (WARNING/2) Definition list ends without a blank line; unexpected unindent.
```